runtime/compiler/runtime/DataCache.hpp

```diff
@@ -41,13 +42,17 @@ class TR_DataCache
    uint8_t         *_allocationMark; // used if we want to give back memory up to previously set mark
    //TR::Monitor       *_mutex;     // Is this needed?
    int32_t          _status;    // mostly RAS at this point
+   OMR::RSSRegion  *_rssRegion; // to report footprint
 public:
    enum {
       RESERVED=1,
       ACTIVE,
       ALMOST_FULL,
       FULL,
    };
+
+   // Note: the constructor is not called
+   TR_DataCache() {}
```

Comment:
What is the purpose of adding this empty constructor?

Reply:
That was more for the comment. To make it clear that the constructor is not called. But I guess I can remove the constructor and just leave the comment.

Reply:
Removed the constructor, but left the comment.